From b3f7ef7b9423dbb5ed1de64d0cfd05b79de1b145 Mon Sep 17 00:00:00 2001
From: Rob Evers <revers@redhat.com>
Date: Mon, 23 Jun 2014 21:41:53 -0400
Subject: [scsi] lpfc: remove self-assignments

Message-id: <1403559726-20488-12-git-send-email-revers@redhat.com>
Patchwork-id: 84430
O-Subject: [RHEL7.1 e-stor PATCH 11/24] scsi: lpfc: remove self-assignments
Bugzilla: 1088574
RH-Acked-by: Ewan Milne <emilne@redhat.com>
RH-Acked-by: Tony Camuso <tcamuso@redhat.com>

From: Dick Kennedy <rkennedy@Emulex.Com>

Submitted by James Smart James.Smart@emulex.com

As spotted by coverty/Dave Jones, removing unnecessary self assignments.

-- james s

Signed-off-by: James Smart james.smart@emulex.com

http://marc.info/?l=linux-scsi&m=139292414129987&w=2
Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/drivers/scsi/lpfc/lpfc_hbadisc.c b/drivers/scsi/lpfc/lpfc_hbadisc.c
index 5aeceb6..59b51c5 100644
--- a/drivers/scsi/lpfc/lpfc_hbadisc.c
+++ b/drivers/scsi/lpfc/lpfc_hbadisc.c
@@ -6189,10 +6189,6 @@ lpfc_read_fcf_conn_tbl(struct lpfc_hba *phba,
 
   memcpy(&conn_entry->conn_rec, &conn_rec[i],
    sizeof(struct lpfc_fcf_conn_rec));
-  conn_entry->conn_rec.vlan_tag =
-   conn_entry->conn_rec.vlan_tag;
-  conn_entry->conn_rec.flags =
-   conn_entry->conn_rec.flags;
   list_add_tail(&conn_entry->list,
    &phba->fcf_conn_rec_list);
  }
-- 
1.7.1